### DISCUSSION TITLE
[POLL] Why & how did you choose AzerothCore?

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3913] [General]
I would like to hear from the community your opinion about AC and the way you ended up choosing it.

Please answer the following questions:

### 1) Why you like AzerothCore?

E.g. what are the features that you appreciate the most, why you prefer it over other projects, why would you recommend it to someone else, etc..

### 2) Where did you find AzerothCore?

E.g. where did you hear about AzerothCore for the first time

**Comment by pangolp** [comment]
**1) Why you like AzerothCore?**
The reasons why I like azerothcore are several, the first of them, is that I feel part of a community of developers and although my contributions are not usually very significant, I like to collaborate because as I use the core for a project of my own, the changes I make, but the changes made by the community, not all of us have to focus on the same issues.

Other reason why I like azerothcore, is that from the first moment I said I wanted to contribute, until the moment I could start doing so, many community members taught me tools like git, and answered all the questions I raised.

**2) Where did you find AzerothCore?**
I came to azerothcore, by chance, some people in a discord, had used the core in a personal way to test things that in the server where I played didn't work and then, I decided to try it. When I saw the big differences there were, I wanted to rent a machine and put a project of my own with this core.

The main difference between azerothcore and the server where I was, is that it works with a version of trintitycore, but it was from 2014 I think, enclosing and not sharing with the rest of the community the arrangements. That way, they became very outdated. My idea, was to contribute with a community and this as updated as possible of the changes that are made, for just, I do not happen the same.

Remember that I use a translator to communicate, it may be that some words are not understood at all well.

**Comment by milestorme** [comment]
1) Why you like AzerothCore?
I actually was closed off to the idea of changing cores i had learnt TrinityCore its a good core, but some of my players wanted custom scripts that i really had a lot of trouble implementing over there, I was a little tipsy one friday and after months of telling people no we are using ?Trinity i swallowed my pride and decided to give it a go, I had previously tryed to compile on my hosted linux server and failed because i didnt know the correct cmake command, but after trying Stygianis the best repack , ide never seen such awesome scripts so stable , not modifieing the core, modules is the greatest idea since sliced bread , so i spent the next week putting it together transfering over my characters base using the AzerothCore Tools and guides and it worked so smoothly and to top everything off there's a community discord that's actually quite helpful,i find myself checking it everyday to learn something new its like going black ill never go back. I really could go on and on the development team the contributors it feels like home. very very stable and lots more working quests
2) Where did you find AzerothCore?
I googled in the beginning and was convinced by my friends to look into it and finally Stygian's repack sold me on it. Big thank you to all of the team and contributors you have done an awesome job rep++

**Comment by ZeegCL** [comment]
1) Why you like AzerothCore?

The main reasons are: most of the contents are fixed, the module support is nice and the members of the dev community are approachable (not like TC).

2) Where did you find AzerothCore?

A friend of mine told me about this project when we were searching for alternatives for our server.

**Comment by ghost** [comment]
**1) Why you like AzerothCore?**
- modules... I like the idea of changing the gameplay by adding different modules
- most of the quests, dungeons, NPCs etc. seem to work very well
- great community

**2) Where did you find AzerothCore?**
Concerning WoW I played vanilla for a few months, TBC and WotLK excessively and quit during Cata. Out of nostalgia I stumbled upon a vanilla repack from conan513 in October 2017, but quickly remembered how tedious vanilla actually was. As WotLK was the peak of WoW in my opinion I tried another repack from him using AC and was quite happy with it. But there were a few issues with quests and NPCs so I took interest in fixing these issues and also wanted to learn how the server works. This lead to contributing a few PRs and finally joining the team. I'm still working on my solo-server (a server just for myself) and will contribute every improvement that I implement because I don't want to keep anything of this just for myself.

**Comment by rpstitz** [comment]
I like AzerothCore for many reasons but mainly because it has taught me a lot about video game servers and I want to learn even more.

I found out about AzerothCore using Google, and when I compared the options available AzerothCore just seemed like the easy choice. I have been running it for over a year with almost no issues, and I am very happy to see that the community is still active.

**Comment by ghost** [comment]
Azerothcore was the reason I got into programming in the first place! One day I was raiding Ulduar on a perilous server and came across a lot of bugged content and, I felt like the developers of said server were doing a poor job. But the urge to learn how to fix content only came when I met PolarCookie and he showed me AzerothCore and TrinityCore.

After having bad experiences with TrinityCore community, I decided to give AzerothCore a shot and found the best wow dev community I could ever ask for. It's true that this core is still a few years behind TrinityCore but that's because TrinityCore has always been there and our community is only a few years old.

However I do believe this to be the best chance for someone new to start with wow emulation because the community here is helpful and supportive. Not only that, the staff will never tell you no to something if It actually improves the core! Unlike your other core choices. 

Data doesn't need to be 100% accurate and everyone here knows that is impossible.
We will never say no to a fix just because you didn't sniff retail data for this pull request. As long as it is as accurate as possible, we will be happy for any contribution, and to me, that's what is most important.

**Comment by nolt** [comment]
1. Why?
- modularity
- rock & solid
- very stable
- great support
2. Where?
I found it in Google, googling what projects support 3.3.5a

**Comment by wizzymore** [comment]
I give this a shot as well 😄 

Why AC?
At first, i didn’t thought much of AC when i saw the number of commits and contributors from Trinity, as this was my first open-source experience i didn’t knew what i was going through and what the meaning of a good community was, but, it didn’t took me much on AC to find what a bad one is.

I found AC i think on mangos rumors when i was looking for another open source emulator and really didn’t like arkemu or wowemu, so i digged deeper and, even after 2 or more years later, here i am 😄

**Comment by FatGrandpa007** [comment]
Why AC? It's super user friendly, the support is beyond amazing.
I've worked with Trinity for many years now, and I only had to take a look at AC for a split second to realise this was the way to go.

Can only recommend, I've had nothing but fun and success with AzerothCore.
Their Discord is fantastic as well.
Give it a try, you have no idea what you're missing out on!

**Comment by ghost** [comment]
The story is the following: 

-At the end of March 2021, our beloved and hated Blizzard opened the beta of BC, as always, almost exclusively available only to influencers and 'important people' despite putting the big ad, to everyone, in our battlenet launcher ^^, to create and take advantage of the hype of something so old and that can be enjoyed in greater or lesser quality for so long on alternative servers it led me to get pissed off, later, that anger was transmuted into a wonderful idea that suddenly appeared in my mind.... Why not have my own copy and server of BC, just for me? 

-From that point on, I embarked on a nearly 3-week adventure in which I've been experimenting with versions of BC, WOTLK, and Cataclysm, skimming dangerous boundaries and learning along the way that there are nasty people in the 'repack' scene who take advantage of the desire of users without much knowledge of the subject to install malicious code, cryptominers and others.

-So when I end up bumping into your community and I read your principles and philosophy, I could not feel happier, there will always be people who stand out for contributing to create things and concepts that are worth defending and developing, despite the fact that in the same fields, there will always be people with a perverse mind who try to achieve the opposite of something with good ethical principles.

To finish, say that I am not a developer, not even close, but what I have plenty of, is time and the desire to contribute in what I can to your community and projects. 

For now, since yesterday I have been enjoying my first character in chromiecast, and the sensations are very beautiful !!

Health and Strength for all of you pals, and best wishes ;)

**Reply by FrancescoBorzi** [reply]
welcome @Eldarix ! what a lovely introduction, thanks for writing this!

> what I have plenty of, is time and the desire to contribute in what I can to your community and projects.

that's awesome, we need a lot of help from non-developers too. For example, it would be awesome if you can help us by testing PRs:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**Reply by ghost** [reply]
Just today, after several hours of headaches and trying to understand things I have no idea about, I managed to put together a new and clean version of AzerothCore with docker, and the truth is that it's always a very satisfying joy to get things like this to work :)

I guess from tomorrow when my neural circuits are reestablished I can focus on learning about testing PR, and from there continue exploring what options to contribute I have.

Thanks again pals, and enjoy

**Comment by alsemo** [comment]
Hi great team,

Hope I'm not too late replying to this.

1) Why you like AzerothCore?
Open source! Dockerize! Also AC is using one of the best expansion (as I read from many online sources) which is WotLK!


2) Where did you find AzerothCore?
It was quite an adventure for me. I have a home server running docker which I also locally hosting Minecraft for my 3 kids.
I never played WoW and tried the trial version a few months ago, played less than 2 hours then didn't touch it again.

As my kids were at home 24/7 due to covid, I wanted them to play an adventure/rpg game, importantly together, so they can roam in the virtual world and have freedom there while we're under lockdown.

So I look into WoW again, but thinking that for them to be inside server with many unknown people, I'm not ready for that. That's when I tried searching for any possibility for self-hosting mmo server. And no secret, AzerothCore was (AFAIK) the only self-hosting mmo server I found which not only open source, but was made easy to run with docker! For this I thank you guys!

Me and the kids are now currently at level 13-14, as a total beginner myself, we're learning the game and enjoying it. You guys made it possible for us to have this virtual adventure!

**Comment by inselberg** [comment]
Simple. AC offered a docker image which is ready to use and not only pretending that it should work.

**Comment by TenebrisWoW** [comment]
I chose Azeroth Core because it runs nicely and is relatively bug-free. I was also looking for a way to easily integrate modules.

I can't say where I've heard of it. I did research and looked for exactly the criteria from above. I also tried TrinityCore, but was less satisfied with it than with AC.

**Comment by mrs-th** [comment]
I chose AC, because I want to learn about WoW emulation and this is a truly open source project with great documentation and support.

I found AC through someone recommending me Chromiecraft to play on.

**Comment by virgo77** [comment]
**1) Why you like AzerothCore?**
I was for many years only aware of MaNGOS, and tried to use it because I wasn't happy of what BliBli made with Cataclysm Expansion. So, it was the only core I found.
After being disappointed with this core and the lot of things that were not working (npc movements essentially and DB French), I finally found EasyMaNGOS wich was a little bit more playable. But, it still was not enough for me to play in Azeroth.
I search the web and found some private servers, which were using TrinityCore. So I tried it. It was better as the core was still in active development. And it's after going through the forum I found AzerothCore. Today, I only use it and I'm very happy because basic scripting is working (npc movements for sure (even some of them are still not moving alone, that's why I would to get a little help about this))
I tried some other core like CMaNGOS. But it's not fully working.

As I'm alone on my WotLK Server, I can't play on dungeons or raid, because I don't know how to try multi-boxing and if some softs can be free (ISBoxer is not, and seems to be really full of functionalities, but too complicated for me. I need something easy to use, and to try) , so I can't tell what is working or not. And I tried to create 255 characters but their stats are not fully usable (HP, mana, etc.)


**2) Where did you find AzerothCore?**
On a TrinityCore forum. I don't really remember and can't be totally sure of that. But today, I only use AC on a Windows Server 2022, as I tried before Debian and older Windows Server.
So, now, each week, i pull an update from GitHub, compile again, search in the worldserver.conf.dist file for some changes, and adapt mine to be closer with the .dist file. And it runs like a charm.
Today I would like to learn a little on how the core is ompiled and how I can adjust some parameters to make the core more playable (to me), but I don't know what is C++ and how to make some little dev on my own. Because I want to adjust the numbers of deposits that are too much in some zones, and some beasts or others that are too much too. So, the BlizzLike experience is really good with AC, but not totally. It's my own opinion :)

Hope this answer will be good for you

V.

**Comment by ghost** [comment]
I hope adding a comment to this isn't frowned upon due to its age. If so, I apologize and wish to have it removed.

**1) Why you like AzerothCore?**

- This has to be one of the quickest and easiest cores to spin up for friends, family and testing. It's also one of the easiest to keep up to date. I'll weekly (or more depending on commits) pull from git and rebuild. 
- Active project with almost daily commits
- Modules and tools (Keira3) are kept up to date as much as possible 
- Very playable and enjoyable out of the box
- Posting a bug report or other issue isn't scary. Project maintainers are friendly and welcome bug reports and are patient with those of us who can report bugs but not necessarily provide a fix. Personally my SQL knowledge is entry level and C/C++ level even worse, but I've always felt welcome when reporting a bug. I've seen this friendly response many times in other bug reports ... even those that don't include the requested information. Thank you for remembering the human behind the github issue 💯 

**2) Where did you find AzerothCore?**
I don't remember. I used TrinityCore for awhile and maybe saw a cherrypick from AC to TC. Or maybe it was talk on wowservers subreddit. 

Thanks for all the work over the years.

**Comment by blinkysc** [comment]
I was looking for a fun project to help teach my kids about open source as well as command line etc. it’s turned into such fun times with me and my kids questing together in Azeroth! Thanks to all the dedicated engineers and supporters!

**Comment by abiaozsh** [comment]
2) Where did you find AzerothCore?
I have found AC from chatgpt, before AC I have tried TC(335), TC is still not perfect, a lot of minor bugs.

1) Why you like AzerothCore?
It's perfect, I can't tell the difference between AC and official server that I have remembered.
With bot module, the experience is even better than the official server.

By the way, is there any plan of development of newer version (cata, MOP etc.) ? other branch(skyfire etc.) is far from playable.

**Comment by Coolsabsab** [comment]
Huge thank to AC team for amazing work u did.
I have chosen AC because it's free, open, community supported, it's working, easy to setup and manage, and today's gold standard, in my opinion.
I found AC simply by searching google

**Comment by OnlyLinux71** [comment]
I started out many years ago on mangos. Operated a vanilla server and had about 150 players. It was fun!! Being in the Military i didn't have time to invest in moving forward to tbc which was the current retail at the time with wotlk in the works. Anyway after 20 + years of government service 13 of which was military I got out in July 2000. Had a horrible experience on the 38th parallel being one of 2 survivors of a 5 man squad and the aftermath of finding out the ones who we encountered were as young as my children at the time back home changed me in an instant. I felt shame and all I wanted to do was go home and left the only life I knew behind and did just that. We both did!! He ended up a police office in CA and I went back home to IN and became rooted ino the food service industry. Cooking became the way I coped with my mental health and I regained my will to live because coming back home wasn't a quick fix. Took awhile to look family and friends in the face. Well, my sons were gamers and actually had my wife onboard. So I guess its been like 20+ in the gaming industry now for me. it was like they were in their own little worlds on these games. I didn't get it at the time but wanted desperately to re-bond so I got into gaming. Remember gaming on dial up right? Anyway so broad band had came out and the boys were on World of Warcraft before TBC. I wasn't sure still what the fuss was about but the night they stood in this massive line for an xpac of TBC until past midnight. Gosh the computers needed upgraded and that was my ticket back in the circle :) Kids got home the next day from school and had brand new computers. I was the bomb!! Then it seemed like every other week the internet was down. SO,, I just did an online query on how to play the game without internet and BAM,, found mangos. Maybe it was cmangos,, dont recall. Started out there. Boys helped a lot letting me know what was broken etc etc and was a good way to bond. The restaurant took off really well and I found myself once again not having a lot of time to progress the server into other xpacs. Opened a gift the boys sacrificed some allowance for to buy me and it was wotlk. So i began playing wow again and loved it. Got 2's and 3's achieves with the boys and it really helped me really recover well. Then I was able to do 2 realms a classic and a tbc realm. was messing with the cmangos and arcemu. Was porting my scripts to arcemu and just stumbled upon someone getting flamed  on a gaming site asking about trinitycore. Of course I had to check it out and quickly moved everything to that project. I am thinking it was around 2007 i got really into emulation was operating a 2 realm wow server I had dumped classic and was doing tbc and wrath but i think cata was the current retail at the time. Eventually dumped tbc and had wrath and cata. Rage quit retail with Cata,,, actually quit pre patch,,, hated the idea of cata. I went all in on private servers. learned from all sorts of ppl!!! treetree, winkdog, wilson212, lots of folks,, those were a few. Closed the restaurant and moved the family to Clearwater, Florida. Became a minister there and and was lead to come back in 2011 back to the place you could experience all 4 seasons in a week,, sometimes in a day :) I had a temporary conviction in my heart and stopped emulation and put all my focus into Linux. Four years ago i developed my own Arch Linux Distro. Just hated windows,,, STILL DO and wont run it period. ALL MY CODING was learned on Windows so the coding was different for games, not drastic just finding alternatives was the main challenge. When I got back into gaming I found Azerothcore and it just has been the absolute best experience ever for community and stability and it just out performs in my opinion. the even better part is I don't even code much anymore,,, ya don't gotta in Azerothcore. With the mods its just an easier. All my scripts soemone has just done better in azerothcore. I dont mind eating crow its true. I was even recently ticked off on their discord page, aint been back sense but its a great community!!!!!! I learned the fine art of detachment years ago and the person who responded was just doing their jobs as a moderator ,,, I guess :) Anyway I now have just a wotlk with tons of modules. Lost a stock pile of scripts on mega, filebeam and mediafire and those platforms refuse to let me reset my password, I had them all started with the old email that i lost access to several years ago. With azerothcore,,, even if i never recover the scripts many of the exist on azerothcore already better anyway, I just liek to help ppl. I like doing things with no support and recently added NixOS support for Azerothcore. NOBODY bit on my post which is crazy cuz my NixOS out performs Debain and windows and is crazy stable. I dont have an issue with authority it was the backbone of what kept me alive in the field before BUT I will ALWAYS post on the wrong forum because I am researched out man lol. I did a search before my recent discord fiasco and it has upwards of 200 posts,,, NOPE not gonna go thru them all but the 50 or so didnt match my criteria so I made the post. it wasnt a bug like they thought i was experiencing it was actually about their own wiki on Azerothcore about quest_template and quest_template_addon. NEITHER mention a startitemcount  but just a startitem. trying to get back into coding I started off making a custom set of items to ARAC module that worked for classs race combos. Aint really a need for a mod i found for a staring zone teleporter that lets you chose the starting zone when ya cant do the quests for the class. I edited the racemask and added item scrolls for things like gift of the naaru, arcane torrent etc etc that lets you perform the quest functions with the item. Works great but im the type that regardless if an error impacts the server i dont want them anyway!!!!  I get the  startitemcount error but i see no startitemcount field ANYWHERE,,, maybe im blind witht he old age of 55 :) Later all or i wont stop typing,, and bad at that!!! English isnt a strength!!
